.github/workflows/qa.yml

```diff
@@ -24,32 +24,42 @@ on:
         description: "Git branch to use"
         required: true
         default: "main"
```

Comment:
Is the qa.yml workflow setup to run with every release? I see it runs from `main` branch, wondering if it should not run from the release tag.